standalone-metastore/src/main/sql/mysql/hive-schema-3.0.0.mysql.sql

```diff
@@ -1100,6 +1101,62 @@ CREATE TABLE `SCHEMA_VERSION` (
   KEY `UNIQUE_VERSION` (`SCHEMA_ID`, `VERSION`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+CREATE TABLE `SCHEMA_VERSION_STATE` (
+  `SCHEMA_VERSION_STATE_ID` BIGINT NOT NULL,
+  `SCHEMA_VERSION_ID` BIGINT NOT NULL,
+  `STATE_ID` BIGINT NOT NULL,
+  `SEQUENCE` INT NOT NULL,
+  `TIMESTAMP` BIGINT NOT NULL,
+  `DETAILS` BLOB,
+  PRIMARY KEY (SCHEMA_VERSION_ID, STATE_ID, SEQUENCE),
+  UNIQUE KEY (id)
+);
+CREATE TABLE `SCHEMA_FIELD` (
+  `SCHEMA_FIELD_ID` BIGINT primary key,
+  `SCHEMA_INSTANCE_ID` BIGINT NOT NULL,
+  `TIMESTAMP` BIGINT NOT NULL,
+  `NAME` VARCHAR(256) NOT NULL,
+  `FIELD_NAME_SPACE` VARCHAR(256),
+  `TYPE` VARCHAR(256) NOT NULL,
+  FOREIGN KEY (SCHEMA_INSTANCE_ID) REFERENCES SCHMEA_VERSION(SCHEMA_VERSION_ID) ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+CREATE TABLE `SCHEMA_SERDES` (
```

Comment:
Can a single schema version have multiple serdes?  SCHEMA_VERSION has a SERDES_ID column in it that connects it to a serde.  This table only seems necessary if a single version can be connected to multiple serdes.